### ISSUE TITLE
Megacures broken with Ageless 4.20

### ISSUE THREAD

**Issue by stevecotton** [TeamSurvivalXtreme/SXCollection#42]
ABILITY_CURES isn't changing to megacures

```
git grep 'define ABILITY_CURES'
Ageless_Era/data/core_override_end.cfg:#define ABILITY_CURES
Ageless_Era/data/core_override_start.cfg:#define ABILITY_CURES
```

**Comment by stevecotton** [comment]
I'm going to do a release with the fix and just local testing. It seems a sufficiently bad bug to require that.

**Comment by stevecotton** [comment]
Fixed in de162a4e0139807b2d3ec481f0f42c1e6272410a
